wireless_modules/middle_module/boot.py

```diff
@@ -0,0 +1,58 @@
+# Author: Kunj
+# Last updated: 1/08/2020
```

Comment:
Don't need these two lines. We already use github to version control so we can already see this data. Would also be annoying to change every time.